### PR TITLE
Add npm on README.md

### PR DESCRIPTION
**Pull request by vsn4ik**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +23,18 @@ Please note: It might take a few hours until a new version is available on cdnjs
 
 ## Development
 
-Once you cloned the repo, you'll need to install [grunt](http://gruntjs.com/) and the development dependencies using [yarn](https://yarnpkg.com/).
+Once you cloned the repo, you'll need to install [grunt](https://gruntjs.com/) and the development dependencies using package manager:
```
Comment (acrobat) [inline README.md]: ....using **a** package manager...

File: README.md [hunk 2]
```diff
@@ -21,7 +23,18 @@ Please note: It might take a few hours until a new version is available on cdnjs
...
 
-    $ [sudo] yarn global add grunt-cli
-    $ yarn install
+* [yarn](https://yarnpkg.com/):
```
Comment (acrobat) [inline README.md]: Add `(Recommended)`  at the end of this line. As we want to encourage people to use yarn. As it is faster, safer to use

Comment (vsn4ik) [inline README.md]: Done.

File: README.md (hunk 1 above)
Comment (vsn4ik) [inline README.md]: Done.

**Comment by acrobat** [comment]
Thanks @vsn4ik!

**Pull request merged**